docs/installation-guide/quickstart.rst

```diff
@@ -0,0 +1,345 @@
+    kube-system   coredns-6b9cb79bf4-jbtxc                      1/1       Running   0          18m       10.233.0.2     bootstrap.novalocal   <none>
+    kube-system   coredns-6b9cb79bf4-tdmz8                      1/1       Running   0          18m       10.233.0.4     bootstrap.novalocal   <none>
+    kube-system   etcd-bootstrap                                1/1       Running   0          17m       172.21.254.7   bootstrap.novalocal   <none>
+    kube-system   kube-apiserver-bootstrap                      1/1       Running   0          17m       172.21.254.7   bootstrap.novalocal   <none>
+    kube-system   kube-controller-manager-bootstrap             1/1       Running   0          17m       172.21.254.7   bootstrap.novalocal   <none>
+    kube-system   kube-proxy-mwxhf                              1/1       Running   0          18m       172.21.254.7   bootstrap.novalocal   <none>
+    kube-system   kube-scheduler-bootstrap                      1/1       Running   0          17m       172.21.254.7   bootstrap.novalocal   <none>
+    kube-system   metalk8s-ui-656f6857b-cdt5p                   1/1       Running   0          18m       10.233.0.3     bootstrap.novalocal   <none>
+    kube-system   package-repositories-bootstrap                1/1       Running   0          17m       172.21.254.7   bootstrap.novalocal   <none>
+    kube-system   registry-bootstrap                            1/1       Running   0          17m       172.21.254.7   bootstrap.novalocal   <none>
+    kube-system   salt-master-bootstrap                         2/2       Running   0          17m       172.21.254.7   bootstrap.novalocal   <none>
+
+
+Adding a master to the cluster
+------------------------------
+
+Now it's time to add more node to the cluster. First you need to secure the
+cluster by adding 2 nodes with etcd and master roles.
+Here is the procedure to add one, simply do it twice to have 3
+masters (bootstrap + 2 new master).
+
+1. Copy the ssh-key to the new node
+
+   .. code-block:: shell
+
+    $ ssh-copy-id <IP-of-the-new-master-node>
+
+2. Create a yaml config file for this new master.
+
+   .. code-block:: yaml
+
+    apiVersion: v1
+    kind: Node
+    metadata:
+      name: <new-master-node-name>
+      annotations:
+        metalk8s.scality.com/ssh-key-path: /etc/metalk8s/pki/id_rsa
+        metalk8s.scality.com/ssh-host: <IP-of-the-new-master-node>
+        metalk8s.scality.com/ssh-sudo: 'false'
+      labels:
+        metalk8s.scality.com/version: '2.0'
+        node-role.kubernetes.io/master: ''
+        node-role.kubernetes.io/etcd: ''
+    spec:
+      taints:
+      - effect: NoSchedule
+        key: node-role.kubernetes.io/master
+      - effect: NoSchedule
+        key: node-role.kubernetes.io/etcd
+
+3. Declare the new master node into the cluster.
+
+   .. code-block:: shell
+
+    $ kubectl --kubeconfig /etc/kubernetes/admin.conf apply -f new-master-node.yaml
```

Comment:
`--kubeconfig` is not needed if you did the `export KUBECONFIG` first.

Reply:
I would lean towards leaving `--kubeconfig` everywhere, and explain at the `export KUBECONFIG` step that it negates the need to have this option.

That way the commands can stand alone without prior expectations.